unfoldingWord/translationCore
# Prevent bookmarks from toggling when they aren't supposed to

#### This pull request addresses:
Addresses #1050 



#### How to test this pull request:
Try and bookmark a check. Switch checks twice to a different check. That check should not be automatically bookmarked (it was before).


---
This change is [<img src="https://reviewable.io/review_button.svg" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/unfoldingword-dev/translationcore/1054)
